test/suites/test_interval.py

```diff
@@ -142,11 +174,263 @@ def test_interval_positional_init(self):
+            # https://github.com/tarantool/tarantool/issues/8878
+            'tarantool': f"datetime.interval.new({{min=-{int(MAX_MIN_RANGE)} + 1}}) - "
+                         r"datetime.interval.new({min=1})",
+            'str': 'tarantool.Interval(year=0, month=0, week=0, day=0, hour=0, '
+                   f'minute=-{int(MAX_MIN_RANGE)}, sec=0, nsec=0, adjust=Adjust.NONE)',
+            'tarantool_8887_issue': True,
+        },
+        'max_minute_interval': {
+            'python': tarantool.Interval(minute=int(MAX_MIN_RANGE)),
+            'msgpack': (b'\x02\x05\xcf\x00\x00\x05\xa0\x07\x91\x6c\x28\x08\x01'),
+            # Reason why Tarantool datetime is so weird here:
+            # https://github.com/tarantool/tarantool/issues/8878
+            'tarantool': f"datetime.interval.new({{min={int(MAX_MIN_RANGE)} - 1}}) + "
+                         r"datetime.interval.new({min=1})",
+            'str': 'tarantool.Interval(year=0, month=0, week=0, day=0, hour=0, '
+                   f'minute={int(MAX_MIN_RANGE)}, sec=0, nsec=0, adjust=Adjust.NONE)',
+            'tarantool_8887_issue': True,
+        },
+        'min_int32_minute_interval': {
+            'python': tarantool.Interval(minute=-2147483648),
+            'msgpack': (b'\x02\x05\xd2\x80\x00\x00\x00\x08\x01'),
+            'tarantool': r"datetime.interval.new({min=-2147483648})",
+            'str': 'tarantool.Interval(year=0, month=0, week=0, day=0, hour=0, '
+                   'minute=-2147483648, sec=0, nsec=0, adjust=Adjust.NONE)',
+        },
+        'max_int32_minute_interval': {
+            'python': tarantool.Interval(minute=2147483647),
+            'msgpack': (b'\x02\x05\xce\x7f\xff\xff\xff\x08\x01'),
+            'tarantool': r"datetime.interval.new({min=2147483647})",
+            'str': 'tarantool.Interval(year=0, month=0, week=0, day=0, hour=0, '
+                   'minute=2147483647, sec=0, nsec=0, adjust=Adjust.NONE)',
+        },
+        'min_sec_interval': {
+            'python': tarantool.Interval(sec=-int(MAX_SEC_RANGE)),
+            'msgpack': (b'\x02\x06\xd3\xff\xfe\xae\x7e\x39\xea\xa6\xa0\x08\x01'),
+            # Reason why Tarantool datetime is so weird here:
+            # https://github.com/tarantool/tarantool/issues/8878
+            'tarantool': f"datetime.interval.new({{sec=-{int(MAX_SEC_RANGE)} + 1}}) - "
+                         r"datetime.interval.new({sec=1})",
+            'str': 'tarantool.Interval(year=0, month=0, week=0, day=0, hour=0, '
+                   f'minute=0, sec=-{int(MAX_SEC_RANGE)}, nsec=0, adjust=Adjust.NONE)',
+            'tarantool_8887_issue': True,
+        },
+        'max_sec_interval': {
+            'python': tarantool.Interval(sec=int(MAX_SEC_RANGE)),
+            'msgpack': (b'\x02\x06\xcf\x00\x01\x51\x81\xc6\x15\x59\x60\x08\x01'),
+            # Reason why Tarantool datetime is so weird here:
+            # https://github.com/tarantool/tarantool/issues/8878
```

Comment:
Will these tests break after the issue being fixed in Tarantool?

Reply:
Could you please write it under a first commit with the weird behavior to make it clear?

```we will be able to simply write EXAMPLE after the fix```

Reply:
Added "Replace with X after fix" comments